src/lib/crypto/kimchi_bindings/js/node_js/dune

```diff
@@ -1,29 +1,32 @@
 (library
  (name node_backend)
  (js_of_ocaml
-  (flags (:include flags.sexp))
+  (flags
+   (:include flags.sexp))
   (javascript_files node_backend.js))
- (instrumentation (backend bisect_ppx))
- (preprocess (pps ppx_version js_of_ocaml-ppx)))
+ (instrumentation
+  (backend bisect_ppx))
+ (preprocess
+  (pps ppx_version js_of_ocaml-ppx)))
 
 (rule
  (targets
-   plonk_wasm_bg.wasm.d.ts
-   plonk_wasm_bg.wasm
-   plonk_wasm.d.ts
-   plonk_wasm.js
-   flags.sexp)
+  plonk_wasm_bg.wasm.d.ts
+  plonk_wasm_bg.wasm
+  plonk_wasm.d.ts
+  plonk_wasm.js
+  flags.sexp)
  (deps
   ../../wasm/Cargo.toml
   (source_tree ../../wasm/src)
   (source_tree ../../wasm/.cargo/config)
-  (source_tree ../../stubs)
-  ../../../Cargo.toml
-  (source_tree ../../../proof-systems) )
+  (source_tree ../../../proof-systems))
```

Comment:
doesn't the js backend implicitly depend on `../../stubs`? because building the stubs is needed to auto-generate the  `kimchi_bindings` ml files, which wrap the `external` interface the wasm/js backend provides?
I'm not sure if we need to capture this "dependency" in this dune file though..

Reply:
CI seems to pass so (﻿ ͡° ͜ʖ ͡°)